### DISCUSSION TITLE
Mod Ideas

### DISCUSSION THREAD

**Discussion by masik16u** [masik16u/Mythic-Charms#1] [Ideas]
## 👋 Welcome!
  Here we will discuss your ideas for the Mythic Charms mod

**Comment by kikugie** [comment]
A spawnable structure in the world containing the altar and a chest containing glowstone nuggets, charm base and some random ingredients. Ingredients may be random from the total pool of possible items to give players motivation to combine different items.

**Reply by masik16u** [reply]
Very good idea!

**Comment by Rikaisan** [comment]
a way to disable the experience nuggets in case another mod also has a way to make XP bottles (like the create mod) (or make them packable with 9 of them or lower their frequency, they fill up way too much space)

Also a "Disable charms near a resonance table" option would be nice too so the collector charm doesn't need to be unequipped (or just add a GUI to the table)

**Reply by masik16u** [reply]
Interesting, I will think about it

**Comment by JasdanVM** [comment]
Charm of Reverse Respawning
When the player dies, they respawn somewhat close to the previous place they died, good for returning to your death point, but you have to die to get there, and since you won't be placed on the exact spot, for a period of time you'll be vulnerable and have no food until you get your stuff back.
(keeps the Recovery Compass useful, if there's something that makes them remain after death, such as the mod Spelunkery).

**Reply by masik16u** [reply]
That's an interesting idea, but it doesn't quite fit. Charms encourage you to not die with them equipped but in this situation, you would wear this charm only to die afterward.
